nautobot/core/ui/nav.py

```diff
@@ -195,6 +201,7 @@ def initial_dict(self) -> dict:
             "weight": self.weight,
             "buttons": {},
             "permissions": self.permissions,
+            "querystring": self.querystring,
```

Comment:
Does `initial_dict` need to be updated for `NavMenuItem` too?

Reply:
It wouldn't hurt. Only the NavMenuButton was utilizing the Initial dict during rendering. The logic for NavMenuItem was handled by different code.